portfolio/src/main/webapp/script.js

```diff
@@ -56,9 +56,60 @@ function showSlides(newIndex) {
 
 /** Called when website is loaded. */
 function init() {
+  showCommentForm();
   showComments();
 }
 
+/**
+ * Returns login status including current email adress and login/logout link.
+ * @return {Object} The status object.
+ */
+function getLoginStatus() {
+  const status = fetch('/user')
+      .then((response) => response.json()).then((status) => {
+        return status;
+      });
+  return status;
+}
+
+/**
+ * Displays a login status message
+ * and if the user is authenticated the comment form.
+ */
+async function showCommentForm() {
+  const status = await getLoginStatus();
```

Comment:
This goes in line with the question above - if the "getLoginStatus" is blocking, why do you have an await here?